ga4gh/server/frontend.py

```diff
@@ -279,16 +280,23 @@ def configure(configFile=None, baseConfig="ProductionConfig",
     app.serverStatus = ServerStatus()
 
     app.backend = _configure_backend(app)
-    app.secret_key = os.urandom(SECRET_KEY_LENGTH)
+    if app.config.get('SECRET_KEY'):
+        app.secret_key = app.config['SECRET_KEY']
+    else:
+        app.secret_key = 'super_secret_CHANGE_ME'
```

Comment:
You can't generate a new secret key for every process, so thanks for fixing that. Instead of defaulting to a hardcoded value I'd like to say that `SECRET_KEY` is a required configuration item for OIDC support. Failing here if that is not set is acceptable behavior.